Added missing Geoms to the plot options in the Climatic>Mapping>Map sub dialog

EstherNjeriLiberatta opened the pull request:
Fixes #7494 
@rdstern @africanmathsinitiative/developers This is ready for review.

rdstern commented:
@EstherNjeriLiberatta very well done, that looks great.
I was surprised, though, in the order of the layers.  It had the labels first, then the points and third was the sf.  I expected them to be the other way round.
Then I tried changing the label size to 0.3 and that gave me the following error, see below.  I wonder if that is because of the ordering of the layers?
![image](https://user-images.githubusercontent.com/11226469/170723688-d3e4c2be-4769-483a-810d-618b81710e65.png)

N-thony commented:
> @EstherNjeriLiberatta very well done, that looks great. I was surprised, though, in the order of the layers. It had the labels first, then the points and third was the sf. I expected them to be the other way round. Then I tried changing the label size to 0.3 and that gave me the following error, see below. I wonder if that is because of the ordering of the layers? ![image](https://user-images.githubusercontent.com/11226469/170723688-d3e4c2be-4769-483a-810d-618b81710e65.png)

@EstherNjeriLiberatta thanks for this, is it ready for review?

EstherNjeriLiberatta commented:
@rdstern The error was coming about when you edit into the sub dialogue which was causing the data to be lost in the geom_point or geom_label_repel code. It is now fixed by adding the dataset to the ggplot() function. The order of the geoms is also fixed. This is ready for review.
@N-thony I have fixed the issue, it's now okay. Thanks

rdstern reviewed (changes requested):
@EstherNjeriLiberatta that's good and seems to work fine.
Ideally, the Point Options and Label Repel options would also be included in the drop-down.  
If that is tricky, then we could merge this and consider the extra geoms as an additional issue?

EstherNjeriLiberatta commented:
@rdstern I suggest we open another issue on this and merge this one. Thank you

rdstern reviewed (approved):
@EstherNjeriLiberatta that's good and seems to work fine.
Ideally, the Point Options and Label Repel options would also be included in the drop-down.  
If that is tricky, then we could merge this and consider the extra geoms as an additional issue?

After your comment I am approving and perhaps you could make a new issue of the point about adding the remaining geoms into the pull-down.

lloyddewit commented on instat/dlgClimaticStationMaps.vb:
```diff
@@ -124,14 +124,14 @@ Public Class dlgClimaticStationMaps
     End Sub
 
     Private Sub SetDefaults()
-        clsParamOperator.Clear()
```
Also delete declaration on line 48?

lloyddewit commented on instat/dlgClimaticStationMaps.vb:
```diff
@@ -346,17 +326,44 @@ Public Class dlgClimaticStationMaps
+            End If
+            If Not ucrReceiverShape.IsEmpty Then
+                clsGGplotOperator.AddParameter("scale_shape_manual", clsRFunctionParameter:=clsScaleShapeFunction, bIncludeArgumentName:=False, iPosition:=2)
+            Else
+                clsGGplotOperator.RemoveParameterByName("scale_shape_manual")
```
parameter already removed above. Do we need these lines?

EstherNjeriLiberatta commented:
@lloyddewit I have resolved your comments above. Thank you.

lloyddewit reviewed (approved):
@EstherNjeriLiberatta Thank you, looks good

lloyddewit commented:
@rdstern Some code changes since your last approval. In theory, the functionality should not be affected. If you can test/reapprove, then we can merge. Thanks
